src/gatsby/gatsby-config.js

```diff
@@ -147,8 +149,11 @@ module.exports = {
                     populate: ["isActive", {
                       campingType: ["campingTypeNumber", "campingTypeCode", "isActive"]
                     }]
-                  },                  
+                  },
                 },
+                parkContacts: {
+                  populate: ["parkOperatorContact"]
```

Comment:
I'm pretty sure this isn't working right.  

parkContacts.contactInformation and parkOperatorContact.defaultContactInformation aren't being populated.

Reply:
I couldn't recreate your issue. Did you check at the API endpoint after giving permission to these collections on Strapi? It works on my end through GraphQL. Please let me know how you faced the issue.
 
![Screenshot 2024-07-29 at 11 16 41 AM](https://github.com/user-attachments/assets/e6217aa0-919e-4f3b-8e44-950f05fd7544)
![Screenshot 2024-07-29 at 11 16 51 AM](https://github.com/user-attachments/assets/ca004a2a-6fd6-4577-9cc2-bfc71bf8671d)